sakura_core/doc/logic/CDocLineMgr.cpp

```diff
@@ -134,6 +134,30 @@ void CDocLineMgr::DeleteLine( CDocLine* pcDocLineDel )
 	}
 }
 
+//! 他のCDocLineMgrの内容を末尾に追加する
+//! 元のCDocLineMgrの内容は空になる
+void CDocLineMgr::AppendAsMove( CDocLineMgr& other )
+{
+	if( other.GetDocLineTop() == nullptr ){ return; }
```

Comment:
指摘じゃないです。ほほ趣味ですが、こう書いても同じ意味。
```suggestion
	if (!other.GetDocLineTop()) return; // otherが空なら抜ける
```

Reply:
GetDocLineTop() がもしbool型なら `GetDocLineTop() == false` とせず `!GetDocLineTop()` とするところですが、ポインタ型なので比較演算子にしました。

Reply:
どっちでもいいみたいです。

誤って代入するのを防止する目的で、
定数を左辺に書く
みたいな文化もあります。

```c++
if ( nullptr == xxx() ){
```